### PR TITLE
Auto update

### PR DESCRIPTION
**Pull request by ericgaspar**
- Add auto update Action

### CODE REVIEW

**Comment by kemenaran** [comment]
Neat.

A question: how does the script knows that releases need to be fetched from two different repos?

File: .github/workflows/updater.sh [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+#=================================================
+# FETCHING LATEST RELEASE AND ITS ASSETS
+#=================================================
+
+# Fetching information
+current_version=$(cat manifest.json | jq -j '.version|split("~")[0]')
+repo=$(cat manifest.json | jq -j '.upstream.code|split("https://github.com/")[1]')
+# Some jq magic is needed, because the latest upstream release is not always the latest version (e.g. security patches for older versions)
+version=$(curl --silent "https://api.github.com/repos/$repo/releases" | jq -r '.[] | select( .prerelease != true ) | .tag_name' | sort -V | tail -1)
+assets=($(curl --silent "https://api.github.com/repos/$repo/releases" | jq -r '[ .[] | select(.tag_name=="'$version'").assets[].browser_download_url ] | join(" ") | @sh' | tr -d "'"))
```
Comment (kemenaran) [inline .github/workflows/updater.sh:24]: I think this is gonna fail for arm and arm64 version, which are third-party builds downloaded from another repository (https://github.com/SmartHoneybee/ubiquitous-memory/releases)

Comment (tituspijean) [reply .github/workflows/updater.sh]: It should be handled now. 🙂

**Review by kemenaran — APPROVED** [review]
Let's give a shot at this 👍

**Pull request merged**